Essentials/src/main/resources/messages.properties

```diff
@@ -620,12 +839,32 @@ powerToolsDisabled=\u00a76All of your power tools have been disabled.
 powerToolsEnabled=\u00a76All of your power tools have been enabled.
 powertoolCommandDescription=Assigns a command to the item in hand.
 powertoolCommandUsage=/<command> [l:|a:|r:|c:|d:][command] [arguments] - {player} can be replaced by name of a clicked player.
+powertoolCommandUsage1=/<command> l:
+powertoolCommandUsage1Description=Lists all powertools on the held item
+powertoolCommandUsage2=/<command> d:
+powertoolCommandUsage2Description=Deletes all powertools on the held item
+powertoolCommandUsage3=/<command> r:<cmd>
+powertoolCommandUsage3Description=Removes the given command from the held item
+powertoolCommandUsage4=/<command> <cmd>
+powertoolCommandUsage4Description=Adds the given command to the held item
```

Comment:
missing the powertool append usage (`a:`)

Reply:
`/powertool a:<cmd>` has the same effect as `/powertool <cmd>`

Reply:
```suggestion
powertoolCommandUsage4Description=Sets the poweretool command of the held item to the given command
powertoolCommandUsage5=/<command> a:<cmd>
powertoolCommandUsage5Description=Adds the given powertool command to the held item
```